### PR TITLE
More ola_trigger work

### PR DESCRIPTION
**Pull request by peternewman**
Also miscellaneous other minor clean-up.

### CODE REVIEW

File: tools/ola_trigger/Context.cpp [hunk 1]
```diff
@@ -86,6 +107,16 @@ void Context::SetSlotOffset(uint16_t offset) {
 
 
 /**
+ * @brief Set the universe variable
+ */
+void Context::SetUniverse(uint32_t universe) {
+  ostringstream str;
+  str << static_cast<int>(universe);
```
Comment (nomis52) [inline tools/ola_trigger/Context.cpp]: Don't we have IntToString for this now?

File: tools/ola_trigger/ola-trigger.cpp [hunk 2]
```diff
@@ -192,6 +192,13 @@ int main(int argc, char *argv[]) {
 
   yyparse();
 
+  // set the core context stuff up
+  if (global_context) {
+    global_context->SetConfigFile(argv[1]);
```
Comment (nomis52) [inline tools/ola_trigger/ola-trigger.cpp]: Pull argv[1] into a config_file variable and change line 203 as well.

Comment (peternewman) [inline tools/ola_trigger/ola-trigger.cpp]: Done

File: tools/ola_trigger/Context.cpp (hunk 1 above)
Comment (peternewman) [inline tools/ola_trigger/Context.cpp]: Indeed, updated.

**Comment by nomis52** [comment]
LGTM

File: tools/ola_trigger/Context.cpp [hunk 3]
```diff
@@ -66,12 +72,26 @@ void Context::Update(const string &name, const string &value) {
 
 
 /**
+ * @brief Set the config file variable
+ */
+void Context::SetConfigFile(const string &config_file) {
+  m_variables[CONFIG_FILE_VARIABLE] = config_file;
```
Comment (peternewman) [inline tools/ola_trigger/Context.cpp]: I switched this from creating a new string object, given the Update function above didn't seem to bother @nomis52 .

**Pull request merged**